src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -30,19 +30,28 @@
 #if TARGET_AUDIOUNIT
 #include "aulayer.h"
 #endif
-/*
-#elif TARGET_VST3
-#include "surgeprocessor.h"
-#elif TARGET_VST2
-#include "vstlayer.h"
-#endif*/
-
-//#include <commctrl.h>
+
+#if MAC || WINDOWS
```

Comment:
Should be done in premake5.lua, not here. Is use the standard for us for feature flags?

Reply:
This is the way we staged zoom also. Have a HOST_SUPPORTS_ZOOM (I could do HOST_SUPPORTS_RUNTIME_FONT) and put it in the code; when we got the last host, remove the #ifdef from the code. I could move it to premake but that seems both more icky and less clear to the reader of the code. Mind if I leave it? (Happy to change the name of the variable. USES_RUNTIME_FONTS? HAS_RUNTIME_FONT_SUPPORT? What do you think?)